react/features/filmstrip/components/web/StatusIndicators.js

```diff
@@ -79,7 +77,6 @@ class StatusIndicators extends Component<Props> {
             <div>
                 { showAudioMutedIndicator ? <AudioMutedIndicator tooltipPosition = { tooltipPosition } /> : null }
                 { showVideoMutedIndicator ? <VideoMutedIndicator tooltipPosition = { tooltipPosition } /> : null }
```

Comment:
Why you removed this line? I see condition that allows to disable the element.

Reply:
The moderator icon is something which we will not need. The disabling it its just another way of getting this done. :)